src/engine/Server.cpp

```diff
@@ -826,14 +814,15 @@ CPP_template_2(typename RequestT, typename ResponseT)(
 
   // Log that we are done with the query and how long it took.
   //
-  // NOTE: We need to explicitly stop the `requestTimer` here because in the
-  // sending code above, it is done only in some cases and not in others (in
-  // particular, not for TSV and CSV because for those, the result does not
-  // contain timing information).
+  // NOTE: We need to explicitly stop the `requestTimer` here because in
+  // the sending code above, it is done only in some cases and not in
+  // others (in particular, not for TSV and CSV because for those, the
+  // result does not contain timing information).
```

Comment:
That historic comment does not make sense, as there is no call to a timer here.
I think it can be removed completely as the above code doesn't need an explicit timer stop and therefore doesn't have it anymore.
```suggestion
```